.github/workflows/rubocop.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       # https://github.com/ruby/setup-ruby
       - name: Set up Ruby
-        uses: ruby/setup-ruby@d4526a55538b775af234ba4af27118ed6f8f6677 # v1.172.0
+        uses: ruby/setup-ruby@086ffb1a2090c870a3f881cc91ea83aa4243d408 # v1.172.0
```

Comment:
This comment is probably not accurate anymore? Which version is this?

Reply:
updated! 1.195.0 https://github.com/ruby/setup-ruby/commit/086ffb1a2090c870a3f881cc91ea83aa4243d408